packages/editor/src/components/entities-saved-states/hooks/use-save-entities.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { store as coreStore } from '@wordpress/core-data';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+import { store as noticesStore } from '@wordpress/notices';
+
+const PUBLISH_ON_SAVE_ENTITIES = [
+	{ kind: 'postType', name: 'wp_navigation' },
+];
+
+export default function useSaveEntities( {
+	onSave,
+	entitiesToSkip = [],
+	close,
```

Comment:
This doesn't seem right, but I think it's fine for now and handle in follow ups because it will require more disentanglement of how to properly call, name, etc..

Reply:
What's preventing this hook to be an action instead?

Reply:
Probably nothing. I can check and change.

Reply:
Updated to action